tna/collections/management/commands/fetch_results_pages.py

```diff
@@ -10,7 +12,7 @@
 
 session = requests.Session()
 
-BASE_URL = "https://beta.nationalarchives.gov.uk"
+BASE_URL = os.getenv("BASE_URL")
```

Comment:
Because Django projects have a settings module, it makes more sense to do this sort of thing in there. That way, everything can be overridden by providing an alternative settings file.

Reply:
Thank you Andy. I added the name as TNA_SCRAPER_BASE_URL, so as not to confuse with other base_urls used. and also it aligns with TNA_SCRAPER_EMAIL.